openedx/core/djangoapps/oauth_dispatch/models.py

```diff
@@ -43,3 +49,82 @@ def verify_access_token_as_expired(cls, access_token):
         is set at the beginning of the epoch which is Jan. 1, 1970
         """
         return access_token.expires == datetime(1970, 1, 1, tzinfo=utc)
+
+
+class ScopedApplication(AbstractApplication):
+    """
+    Custom Django OAuth Toolkit Application model that enables the definition
+    of scopes that are authorized for the given Application.
+    """
+    FILTER_USER_ME = 'user:me'
+
+    scopes = ListCharField(
+        base_field=models.CharField(max_length=32),
+        size=10,
+        max_length=(10 * 33),  # 10 * 32 character scopes, plus commas
```

Comment:
Where does this number come from? From the RFC?

Reply:
I didn't see any specification of size limits in the RFC. This was copied over from the original Microsoft PR. Did you have a different number in mind?

Reply:
@mettursathish  Is there a reason your team chose this limit?

Reply:
Bumping this from 10 to 25 as discussed on Slack#architecture.

Reply:
no specific reason why we chose 10.